mq/datatype/_window.lua

```diff
@@ -69,7 +69,12 @@ function window.RightMouseHeldUp() end
 function window.RightMouseUp() end
 
 ---Selects the specified window
-function window.Select() end
+---@param index integer: Index to select
```

Comment:
all params are strings. (anything not a string is implicitly converted to a string. so this should also accept string.

also, i think the colon may be out of place.

Reply:
added